extensions/micrometer/deployment/src/main/java/io/quarkus/micrometer/deployment/MicrometerProcessor.java

```diff
@@ -233,4 +237,27 @@ ReflectiveClassBuildItem createReflectiveBuildItem(DotName sourceAnnotation, Ind
         return ReflectiveClassBuildItem.builder(classes.toArray(new String[0])).build();
     }
 
+    @BuildStep(onlyIf = IsDevelopment.class)
+    public CardPageBuildItem createCard(List<RegistryBuildItem> registries) {
+        var card = new CardPageBuildItem("Micrometer metrics");
+
+        var json = registries.stream().filter(r -> "JSON".equals(r.name())).map(RegistryBuildItem::path).findFirst()
+                .orElse(null);
+        var prom = registries.stream().filter(r -> "Prometheus".equals(r.name())).map(RegistryBuildItem::path).findFirst()
+                .orElse(null);
+
+        if (prom != null) {
+            card.addPage(Page.externalPageBuilder("Prometheus")
+                    .icon("font-awesome-solid:chart-line")
+                    .url(prom));
+        } else if (json != null) {
+            card.addPage(Page.externalPageBuilder("JSON")
+                    .icon("font-awesome-solid:chart-line")
+                    .url(json)
+                    .isJsonContent());
+        }
+
```

Comment:
Is there a reason this is either/or? Both can be present at the same time.

I thought I already had a registry build item of some kind.. because there can be more than that, even, if you wanted to show a list of discovered registries, you could use this existing build item: 

https://github.com/quarkusio/quarkus/blob/main/extensions/micrometer/deployment/src/main/java/io/quarkus/micrometer/deployment/MicrometerRegistryProviderBuildItem.java

Reply:
ok. you can have both the json & prometheus (and other) registries available at the same time. If it doesn't work, that's a bug. (What you can't have is smallrye-metrics & micrometer at the same time)

Reply:
I didn't reuse [MicrometerRegistryProviderBuildItem.java](https://github.com/quarkusio/quarkus/blob/main/extensions/micrometer/deployment/src/main/java/io/quarkus/micrometer/deployment/MicrometerRegistryProviderBuildItem.java) as I need different data (name, path..., I don't really care about the class name).

Let me try to have both registries side by side.

Reply:
Ok... well... got both registry working with:

```
quarkus.micrometer.export.json.enabled=true
quarkus.micrometer.export.json.path=metrics/json
quarkus.micrometer.export.prometheus.path=metrics/prometheus
```

updating the code.